src/vs/editor/common/services/semanticTokensProviderStyling.ts

```diff
@@ -36,8 +40,8 @@ export class SemanticTokensProviderStyling {
 		let metadata: number;
 		if (entry) {
 			metadata = entry.metadata;
-			if (this._logService.getLevel() === LogLevel.Trace) {
-				this._logService.trace(`SemanticTokensProviderStyling [CACHED] ${tokenTypeIndex} / ${tokenModifierSet}: foreground ${TokenMetadata.getForeground(metadata)}, fontStyle ${TokenMetadata.getFontStyle(metadata).toString(2)}`);
+			if (this._logger.getLevel() === LogLevel.Trace) {
```

Comment:
Wondering why you have to check for the log level? Logger already does it for you.